### PR TITLE
Wedeploycom beta

### PR DESCRIPTION
**Pull request by jonnilundy**
Update wedeploy.com to work on Beta infrastructure

### CODE REVIEW

**Comment by jonnilundy** [comment]
Tested here: https://ui-wedeploy.wedeploy.sh/

File: ui/src/components/Feedback.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Feedback extends Component {
 	}
 
 	sendFeedbackToWeDeploy(liked) {
-		return WeDeploy.data('db.wedeploy.wedeploy.io')
+		return WeDeploy.data('https://db-wedeploy.wedeploy.io')
```
Comment (zenorocha) [inline ui/src/components/Feedback.js]: is `https://` needed now? Can't we just use `WeDeploy.data('db-wedeploy.wedeploy.io')`

Comment (jonnilundy) [inline ui/src/components/Feedback.js]: When I asked Eduardo yesterday, that is what he said. Could you verify @eduardolundgren?

**Comment by jonnilundy** [comment]
#215

**Pull request closed without merging**